src/video-player.js

```diff
@@ -115,6 +119,13 @@ class VideoPlayer extends Utils.mixin(Eventable) {
     this._initPlaylistWidget();
     this._initJumpButtons();
     this._setVideoJsListeners(ready);
+    this._sendAnalytics(this.playerOptions);
+  }
+
+  _sendAnalytics(options) {
+    const opts = analyticsGetPlayerOptions(options);
+    const qs = flatten(opts);
```

Comment:
you no longer need to flatten, you just need to create query search params out of the object you have
Also you should change the name opts to maybe `analyticsData`
```suggestion
const queryParams = new URLSearchParams(opts)
```